src/phoenix/experimental/evals/functions/processing.py

```diff
@@ -0,0 +1,51 @@
+from typing import List
+
+from ..models import BaseEvalModel
+
+
+def truncate_text_by_model(model: BaseEvalModel, text: str, token_buffer: int = 0) -> str:
+    max_token_count = model.max_context_size - token_buffer
+    if model.get_token_count_from_text(text) > max_token_count:
+        return (
+            model.get_text_from_tokens(model.get_tokens_from_text(text)[:max_token_count]) + "..."
+        )
+    return text
+
+
+def concatenate_texts(texts: List[str], join_character: str = " ") -> str:
+    # Check if value is a list
+    if not isinstance(texts, list):
+        raise TypeError(f"Expected a list, but got {type(texts)}.")
+    # Check if all elements in the list can be converted to string
+    for text in texts:
+        if not isinstance(text, (str, int, float)):
+            raise TypeError(
+                f"Unexpected data type {type(text)} in the list. Expected str, int, or float."
+            )
+    return join_character.join(map(str, texts))
```

Comment:
Looks like this function is joining a list of strings and doing type checks on the inputs. Doesn't look evals-related. Can we just run `" ".join(text)` wherever this function is being invoked instead, or define the function inside of the script or notebook where it is being invoked?

Not clear on the need for the type checks and the `map(str, texts)` if the input is of type `List[str]`. If the input type is unknown and this function is meant to clean the data and handle edge-cases, we should probably type the input differently, e.g., `List[Any]`.

Reply:
Deleted function, agreed with what you described